src/cargo/core/dependency.rs

```diff
@@ -54,6 +67,16 @@ pub enum Kind {
     Build,
 }
 
+impl Encodable for Kind {
+    fn encode<S: Encoder>(&self, s: &mut S) -> Result<(), S::Error> {
+        match *self {
+            Kind::Normal => "normal",
```

Comment:
For now, this may want to encode as `None` (or null) to mirror what's written down in Cargo.toml